spacy/displacy/templates.py

```diff
@@ -52,6 +52,7 @@
 <mark class="entity" style="background: {bg}; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
     {text}
     <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">{label}</span>
+    <a style="text-decoration: none; color: black; font-weight: bold" href="{kb_url}">{kb_id}</a>
```

Comment:
I was wondering - if we include the KB ID into the `span` element, the formatting of the label and the KB ID will be exactly the same. That seems better?
```suggestion
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">{label}
    <a style="text-decoration: none; color: black; font-weight: bold" href="{kb_url}">{kb_id}</a>
    </span>
```

Reply:
The only disadvantage I see is that there is that there is no "deliminter" between the label and KB ID, but that would kind of complicate the code further. e.g. we could insert a "-" between label and ID, but then only if both are available...

Reply:
KB ID in the `span` element indeed looks better and more compact!